### ISSUE TITLE
Error message while loading the wish log

### ISSUE THREAD

**Issue by DJMo13** [URK96/ResinTimer#23]
![image](https://user-images.githubusercontent.com/42299923/143051878-bcbe813c-23df-46c3-a4c5-0936fa8d7903.png)

It loads and then comes the error even if the feature works because on one of my 100 tries I made it to load the complete log!

**Comment by URK96** [comment]
Thanks to report issue.

After Genshin Impact 2.3 update, miHoYo stop Wish Log API server about 1~2 days for checking status. So we cannot use wish history by this API during these time.

As of today, API server is back to normal. But miHoYo add GET request cycle limit. 
So i adjust interval and apply on next version. I will upload this fix update as soon as possible.

**Comment by URK96** [comment]
This issue is solved on 1.14.2

**Comment by kainili** [comment]
Hello, the issue never has been solve for me. I've been waiting few days as you recommended but it kept giving me the same error as DJMo13 screenshot and I just gave-up on checking. 

Today I've updated your app and could check Mihoyo API status and the 3 categories was failed and still cannot check wish log.

I'm on European server. Don't know if it changes anything.

![Screenshot_20220120-133203](https://user-images.githubusercontent.com/56433935/150340085-de8e9920-9c1f-4269-a4b6-ea4d4c61e797.png)
